nil-docs/zkllvm/getting-started/writing-a-simple-circuit.mdx

```diff
@@ -0,0 +1,88 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+# Writing a simple circuit
+
+This tutorial describes how to create a simple circuit and use elliptic curve fields.
+
+:::tip
+
+The C++ examples in this tutorial and across the rest of the documentation use types and algorithms from [**the `crypto3` library**](https://github.com/NilFoundation/crypto3).
+
+The Rust examples use types from the forks of [**the `arkwors-rs algebra` project**](https://github.com/arkworks-rs/algebra).
```

Comment:
Maybe it would be better to call it "arkworks project" here with this link:
https://github.com/arkworks-rs